### ISSUE TITLE
CARE 2D: Loss curves not displaying

### ISSUE THREAD

**Issue by kaarjel** [HenriquesLab/ZeroCostDL4Mic#335]
Hi, 

I am using CARE 2D on Google Colab and am getting an error after the training has completed. I am also not able to visualize the loss curves of the training. Below is the error I am getting:

![image](https://github.com/user-attachments/assets/11696f57-ba05-4675-8ccf-07df43997cc3)

In the 'Inspection of loss function' cell, the plots are empty:
![image](https://github.com/user-attachments/assets/5495540c-69b9-425f-9ac4-7b3c35ce2192)

Hope you can help. 

Cheers,
Kaarjel

**Comment by moore-cl** [comment]
Hi Kaarjel,

I have had this error myself before, it seems to be because 'lr' is not defined and is referred to as 'learning rate' in line 25. By changing line 25 'learning rate' to 'learning_rate' and then line 27  'lr' to 'learning_rate' so they matched fixed the problem and my curves plot. The creators of the notebook can jump in here if thats not the correct fix.

**Comment by kaarjel** [comment]
Hi moore-cl,

That fix worked, thanks for your help and prompt reply!

Cheers,
Kaarjel
